src/store/getters/state-getters.ts

```diff
@@ -159,8 +159,8 @@ export const getCorrectedFilingType = (state: StateIF): string => {
 }
 
 /** get error correction type (client or a staff error correction). */
-export const getErrorCorrectionType = (state: StateIF): string => {
-  return state.stateModel.tombstone.type
+export const getCorrectionType = (state: StateIF): CorrectionErrorTypes => {
```

Comment:
There is already an enum named CorrectionTypes, so this getter might cause some confusion. I suggest naming this according to the enum it returns.